src/components/ToolbarDropdown/ToolbarDropdown.tsx

```diff
@@ -6,15 +6,17 @@ type ToolbarDropdownProps = {
   disabled: boolean;
   id?: string;
   nameDropdown?: string;
-  initialValue: number | string;
-  initialLabel: string | undefined;
+  value?: number | string;
+  initialValue?: number | string;
```

Comment:
Yes, at least here in this file (and some other places in the same file):
https://github.com/kiali/kiali-ui/blob/85cd32b761cb702f6bf8a7ca7effe003be558d04/src/components/MetricsOptions/MetricsOptionsBar.tsx#L112-L124

Reply:
mmh, my point is, why dont we use the `initialValue` and `initialLabel` instead of adding a new `label` and `value` to the Component?
What's the difference between `initialLabel` and `label`? I think it makes more confusing to the component user.

Reply:
This works like `defaultKey` and `activeKey` of patternfly tab control.
If you set `initial*` properties you dropdowns are _uncontrolled_. You are setting initial values and then the control will decide what to highlight and what to show in the title.
If you set `value` and `label` properties, the dropdowns are _controlled_. This means that the dropdowns won't change it's state (selection and title) unless these properties are updated.

Reply:
Ah, that makes sense, thanks!